lighthouse-core/report/html/templates.html

```diff
@@ -351,6 +351,7 @@
         <a href="#" class="report-icon report-icon--download" data-action="save-json">Save as JSON</a>
         <a href="#" class="report-icon report-icon--open lh-export--viewer" data-action="open-viewer">Open in Viewer</a>
         <a href="#" class="report-icon report-icon--open lh-export--gist" data-action="save-gist">Save as Gist</a>
+        <a href="#" class="report-icon report-icon--open lh-export--dark" data-action="toggle-dark">Toggle Dark Theme</a>
```

Comment:
lets use a real icon for this. 
lightbulb seems good. 💡

Reply:
ok but i dont love how i did it..